### PR TITLE
Created some dtos

### PR DESCRIPTION
**Pull request by Mario2414**

### CODE REVIEW

File: FaceAnalyzer.Api/Business/Contracts/EmotionDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Business/Contracts/EmotionDto.cs:1]: Remove the Reaction entity navigation.
ReactionID => ReactionId

File: FaceAnalyzer.Api/Business/Contracts/ReactionDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Business/Contracts/ReactionDto.cs:1]: Remove Emotiin entity collection
Remove Stimuli entity
Typo in ParticipantName

File: FaceAnalyzer.Api/Business/Contracts/StimuliDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Business/Contracts/StimuliDto.cs:1]: Remove reactions
Remove experiment

File: FaceAnalyzer.Api/Service/Contracts/CreateEmotionDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Service/Contracts/CreateEmotionDto.cs:1]: Remove Id  and Reaction Entity

File: FaceAnalyzer.Api/Service/Contracts/CreateExperimentDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Service/Contracts/CreateExperimentDto.cs:1]: Remove Id

File: FaceAnalyzer.Api/Service/Contracts/CreateReactionDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Service/Contracts/CreateReactionDto.cs:1]: Keep only StimuliId and ParticipantName

File: FaceAnalyzer.Api/Service/Contracts/CreateStimuliDto.cs
Comment (TA50) [inline FaceAnalyzer.Api/Service/Contracts/CreateStimuliDto.cs:1]: Remove entities
Experiment and Reactions

**Review by TA50 — CHANGES_REQUESTED** [review]

**Comment by Mario2414** [comment]
Pushed the new commit, check if it's ok

**Review by TA50 — APPROVED** [review]

**Pull request merged**